README.md

```diff
@@ -27,38 +27,86 @@ currently:
-- Add the necessary bits in to make your pi "headless" (a file named just "ssh" and a "wpa_supplicant.conf" with your wifi credentials) https://www.raspberrypi.org/documentation/configuration/wireless/headless.md
-- Boot the pi and _find it somehow_. Most routers have a page that tells you what IP address everything is using, otherwise there's an app on ios and android called "fing". SSH into it using putty or terminal and set it up. I suggest you run the terminal command "passwd" to change the pi password from the default of 'raspberry' to something else. You can also change stuff in "sudo raspi-config". 
-- I recently discovered that the easiest way to transfer files is by SSH file transfer using something like filezilla. Get filezilla and upload the files for this project just into the home directory /home/pi. Connecting in filezilla is easy so long as you remember it's port 22 (standard FTP is 23 so filezilla tries to default to this!). 
+- Download Respberry Pi Imager from the raspberry pi page https://www.raspberrypi.org/downloads/raspbian/
+- Press "Operating system" then select  "Raspberry Pi OS (Other)" > "Raspberry Pi OS Lite (32 bit)"
+- Choose your storage
+- Click the advanced cog icon, then fill out the info to enable headless boot
+  - You could enter "octoprice" as the hostname
+  - Enable SSH, and enter your public key or a password
+  - Set a username and password
+  - Configure Wireless LAN
+  - Set your locale
+  - Save
+- Press write
 
--You now need to install the libraries using the one line script from pimoroni if you are using one of their displays. At time of writing this page is for inkyphat: https://learn.pimoroni.com/tutorial/sandyj/getting-started-with-inky-phat and the line to run (in an SSH shell on the pi obviously) is 
-```
-curl https://get.pimoroni.com/inky | bash
-```
-This step installs loads of stuff including the PIL libraries and the inkyphat libraries. Nice one pimoroni. It should be noted that it does throw up some errors during and at the end and these seem fine to ignore. 
+Once imaged, put the SD card in your Pi and wait a while for it to boot (it can take 5-10 minutes). It should connect to your wifi, then you can ssh to it using `ssh <username>@<hostname>.local` replacing the username and hostname for those entered when imaging
 
-- The pi is ready for our code (and anything else you want to do with it!)
+Once you have an ssh terminal, you can get started with setting up our project
 
-# How to use the code
+# Installing
 
-We're ready for the bits specific to this project now! 
+- Install the libraries for inky phat using the [one line script from pimoroni](https://learn.pimoroni.com/tutorial/sandyj/getting-started-with-inky-phat)
+
+  ```
+  curl https://get.pimoroni.com/inky | bash
+  ```
+
+- Install git:
+
+  ```
+  sudo apt install git
```

Comment:
the command to pimoroni above installs git near the end of the install. I certainly didn't need to install git specifically on a fresh pi.

Reply:
It's because the next step uses git clone to fetch the project. I don't think git was installed already

Reply:
The git clone worked fine without the explicit install. I spotted that git was part of the pimoroni install so just left it out. 

No big deal, just a step that can be skipped to speed up the process.